src/HumanView.cpp

```diff
@@ -402,23 +402,22 @@ std::string HumanView::formatCountDown(int countDown)
 void HumanView::drawObjects() {
     display.clear();
     display.draw(background);
-    float x = logic.getPlayer().getSprite().getPosition().x;
+    float x = logic.getPlayer().getAnimatedSprite().getPosition().x;
     if ( x < width/2 ) {
         x = width/2;
     } 
-
     view.setCenter(x,height/2);
     display.setView(view);
     for(int i = 0; i < logic.getActors().size(); ++i)
     {
         display.draw(logic.getActors()[i].get().getSprite());
     }
+    display.draw(logic.getPlayer().getAnimatedSprite());
     sf::Text timerText("TIME\n " + formatCountDown(logic.getCountDown()), digitalFont, 40);
     timerText.setLineSpacing(1.5f);
     timerText.setPosition((view.getCenter().x + width/2) - timerText.getGlobalBounds().width - 30.f, view.getSize().y / 60);
     timerText.setFillColor(sf::Color::White);
     display.draw(timerText);
-    display.draw(logic.getPlayer().getSprite());
```

Comment:
Put the display.draw for the player back over here, just so the player is in the foreground while the timer is in the background